src/open_inwoner/configurations/models.py

```diff
@@ -396,6 +396,11 @@ class SiteConfiguration(SingletonModel):
             "The text that should display when OpenId connect is set as a login method"
         ),
     )
+    openid_admin_only = models.BooleanField(
```

Comment:
I'm not 100% sure if the is the most accurate name and field for what this does (conceptually if an 'only' boolean is false then you'd expect it to mean 'both' (and not 'the other' as in this case).

It could be a choicefield with choices for 'admins' and 'users' (with default 'admins', and in the future we could add another choice for both 'admins and users'.

Then for convenience usage in the template add two `@property's on the model, something like `openid_enabled_for_admin` and `openid_enabled_for_users` and use those in the templates.